docs/export/export.md

```diff
@@ -474,7 +474,7 @@ As of June 2024, all function exported with version 9
 
 ```python
 >>> from jax import export
->>> exp: export.Exported = export.export(jnp.cos)(1.)
+>>> exp: export.Exported = export.export(jax.jit(jnp.cos))(1.)
```

Comment:
Does this doctest no longer work without `jit`? If not, is the error message instructive enough?

Reply:
Yes, this no longer works. The object passed to `export.export` must be the output of `jax.jit`, and the error is pretty clear if it's not:
```python
>>> export.export(jnp.sin)
ValueError: Function to be exported must be the result of `jit` but is: <jnp.ufunc 'sin'>
```